docs/run/rest-api-execution-modes.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Execution modes using REST API
+description: How to run a quantum computing job in a Qiskit Runtime session.
+---
+
+# Execution modes using REST API
+
+There are several ways to run your Qiskit Primitive workloads using REST APIs, depending on your needs. The three modes: job, session and batch are explained below. 
+
+
+## Job mode with REST API
+
+In job mode, a single primitive request of the estimator or the sampler made without a context manager. See how to run a quatum circuit using [Estimator](rest-api-primitives#estimator-primitive-with-rest-api) and [Sampler](rest-api-primitives#sampler-primitive-with-rest-api) for some examples. 
+
+## Session mode with REST API
+
+A session is a Qiskit Runtime feature that lets you efficiently run multi-job iterative workloads on quantum computers. Using sessions helps avoid delays caused by queuing each job separately, which can be particularly useful for iterative tasks that require frequent communication between classical and quantum resources. More details about Sessions can be found in the [documentation](https://docs.quantum.ibm.com/run/sessions).
+
+### Start a Session 
+
+You begin with creating a session and obtaining a Session ID. 
+
+```python
+import json
+ 
+sessionsUrl = "https://api.quantum-computing.ibm.com/runtime/sessions"
+ 
+headersList = {
+  "Accept": "application/json",
+  'x-access-token':auth_id,
+  "Content-Type": "application/json" 
+}
+ 
+payload = json.dumps({
+  "backend": backend,
+  "instance": "hub1/group1/project1",
+})
+ 
+response = requests.request("POST", sessionsUrl, data=payload,  headers=headersList)
+ 
+sessionId = response.json()['id']
+ 
+print(response.json())
+```
+Output
+```text
+{'id': 'crw9s7cdbt40008jxesg'}
+```
+
+
+### Close a Session 
+
+It is very good practice to close a `Session` when all jobs are done. This will reduce wait time for subsequent users.
+
+```python
+closureURL="https://api.quantum-computing.ibm.com/runtime/sessions/"+sessionId+"/close"
+
+headersList = {
+  "Accept": "application/json",
+  'x-access-token':auth_id,
+}
+ 
+closure_response = requests.request(
+    "DELETE",
+    closureURL, 
+    headers=headersList)
+
+print("Session closure response ok?:",closure_response.ok,closure_response.text)
+```
+Output
+```text
+Session closure response ok?: True 
+```
+
+
+## Batch mode with REST API
+
+Alternatively, you can also submit a batch job by specifying the `mode` in the request payload.
```

Comment:
This intro sentence feels a bit short compared to the one about sessions. I like how in the sessions intro you included a link to the other session docs, can you do the same here for batch?

Reply:
Sure